client/src/rpc_config.rs

```diff
@@ -28,6 +28,8 @@ pub struct RpcSendTransactionConfig {
 pub struct RpcSimulateTransactionConfig {
     #[serde(default)]
     pub sig_verify: bool,
+    #[serde(default)]
+    pub use_most_recent_blockhash: bool,
```

Comment:
Maybe this?
```suggestion
    pub replace_recent_blockhash: bool,
```

"most recent blockhash" reads a rough in my brain :)

Reply:
works for me